de.metas.business/src/main/java/de/metas/order/IOrderLineQuickInputValidator.java

```diff
@@ -0,0 +1,31 @@
+package de.metas.order;
+
+import de.metas.bpartner.BPartnerId;
+import de.metas.product.ProductId;
+
+/*
+ * #%L
+ * de.metas.business
+ * %%
+ * Copyright (C) 2018 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+public interface IOrderLineQuickInputValidator
+{
+	void validate(BPartnerId bpartnerId, ProductId productId);
```

Comment:
* please document what an implementor should do if something is not valid
* why not return a `ValidationResult` or something? benefits
  * >1 invalid-reasons could be collected and shown to the user
  * one could use it to filter out invalid combinations, to not even show them to the user.

Reply:
Done